### ISSUE TITLE
Make DEFAULT_buy_max_change_percentage relative to 24h market change

### ISSUE THREAD

**Issue by loopy83** [taniman/profit-trailer#1768]
Hi,

DEFAULT_buy_max_change_percentage = x
DEFAULT_buy_min_change_percentage = y
and 
DEFAULT_DCA_buy_max_change_percentage = x
DEFAULT_DCA_buy_min_change_percentage = y

are really strong settings for a good P&D protection... BUT!

Please make these settings relative to 24h market change.

So 24h-market-change + DEFAULT_buy_max_change_percentage = SOM criteria
So 24h-market-change - DEFAULT_buy_min_change_percentage = SOM criteria

Normally I used values aroun 7-10% for several month now using an external scirpt. I never bought a P&D again during that time. Now this great feature is integrated in PT-V2, but I think it need liilte fix.

When market is 10% up and many many coins are 8-15% up, you will send many coins to SOM. So you will miss massive bull run when buying no coins.

This request is based on personal experience since Nov'17. My external script did a VERY GOOD job so I would really appreciate this little fix for this feature.

**Comment by loopy83** [comment]
I shared this script in German Community... its nothing special.
Just small script getting API 24h change for every coin, calculate 24h average change of market and then switching to SOM = true or false for every coin in PAIRS. Its written in python.

But since its not supported in V2 anymore because I dont have access to config files I see no point to share it.

I also did some more stuff like dynamic trailing value depending on number of sales and a market drop trigger to switch on global SOM.

If you're still interested... PM me @BenchBros
